dns-dodo.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=The dns-dodo DNS record updater
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/dns-dodo update-dns -p
```

Comment:
lol, noticed you changed from your home dir. Current readme instructions say how to build which ends up in $GOPATH/bin Maybe add a comment to this file to update the path or copy $GOPATH/bin/dns-dodo to /usr/bin?